cvmfs/mountpoint.cc

```diff
@@ -1395,7 +1401,22 @@ bool MountPoint::CreateTracer() {
       boot_status_ = loader::kFailOptions;
       return false;
     }
-    tracer_->Activate(kTracerBufferSize, kTracerFlushThreshold, optarg);
+    int tracebufferSize = kTracerBufferSize;
+    int tracebufferThreshold = kTracerFlushThreshold;
+
+    string tracebufferSizeOpt;
```

Comment:
For consistency, should be changed to reuse `optarg` for the options values.  The previously stored trace file name should then be saved in a separate variable.